### PR TITLE
fix: fix descriptions demo warning

### LINKED ISSUE #18057: components/descriptions/demo/border.md  console报错
- [ ] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
[https://github.com/ant-design/ant-design/edit/master/components/descriptions/demo/border.md](https://github.com/ant-design/ant-design/edit/master/components/descriptions/demo/border.md)

### Steps to reproduce
25    <Descriptions.Item label="Usage Time" span={3}>

25行  span应该是2

### What is expected?
25行  span={2}

### What is actually happening?
span={3}

| Environment | Info |
|---|---|
| antd | 3.20.7 |
| React | react |
| System | windows |
| Browser | chrome |

### PR DESCRIPTION
**Pull request by chenshuai2144**
### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 💡 Background and solution

好像是测试忘记删了
close https://github.com/ant-design/ant-design/issues/18057

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |       无需    |
| 🇨🇳 Chinese |         no need   |

### ☑️ Self Check before Merge

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

-----
[View rendered components/descriptions/demo/border.md](https://github.com/ant-design/ant-design/blob/fix-error-descriptions-demo/components/descriptions/demo/border.md)

### CODE REVIEW

File: components/descriptions/demo/border.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ ReactDOM.render(
     <Descriptions.Item label="Billing Mode">Prepaid</Descriptions.Item>
     <Descriptions.Item label="Automatic Renewal">YES</Descriptions.Item>
     <Descriptions.Item label="Order time">2018-04-24 18:00:00</Descriptions.Item>
-    <Descriptions.Item label="Usage Time" span={3}>
+    <Descriptions.Item label="Usage Time" span={2}>
       2019-04-24 18:00:00
     </Descriptions.Item>
     <Descriptions.Item label="Status" span={3}>
```
Comment (afc163) [inline components/descriptions/demo/border.md]: 这个也是 2 吧

**Comment by afc163** [comment]
image

这些说明呢。

**Comment by afc163** [comment]
还需要更新 snapshots

**Comment by afc163** [comment]
rebase master

**Comment by chenshuai2144** [comment]
https://github.com/ant-design/ant-design/pull/18270  在这个pr里面。

File: components/descriptions/demo/border.md (hunk 1 above)
Comment (afc163) [reply components/descriptions/demo/border.md]: 这里

Comment (chenshuai2144) [reply components/descriptions/demo/border.md]: 默认一共有三列，只是上面有四个，所以要设置为2

Comment (afc163) [reply components/descriptions/demo/border.md]: 👌

**Pull request merged**